probe/docker/container.go

```diff
@@ -220,6 +222,48 @@ func (c *container) ports(localAddrs []net.IP) report.StringSet {
 	return report.MakeStringSet(ports...)
 }
 
+func (c *container) cpuPercentMetric() report.Metric {
+	result := report.MakeMetric()
+	if len(c.pendingStats) < 2 {
```

Comment:
We should probably still return the sample (if we have one), so that we can write out the "latest value", but not draw a sparkline, I guess.